cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,49 @@
+import HomeAndCataloguePageObject
+from '../support/pages/homeСatalogue.pageObject';
+import CartPage from '../support/pages/cartPage.pageObject';
```

Comment:
where is this file?

Reply:
support/pages

Reply:
put it doesn't exist in the pull request, I can't run your tests without this file